docs/developer/serving/services/custom-domains.md

```diff
@@ -7,6 +7,10 @@ You can create a `DomainMapping` object to map a single, non-wildcard domain to
 For example, if you own the domain name `example.org`, and you configure the domain DNS to reference your Knative cluster, you can use domain mapping to
 serve a Knative Service at this domain.
 
+!!!note
+   If you create a domain mapping to map to a [private Knative Service](../developer/serving/services/private-services.md),
+   the private Knative Service will be accessible from public internet with the custom domain of the domain mapping.
+
```

Comment:
```suggestion
!!! note
    If you create a domain mapping to map to a [private Knative Service](../developer/serving/services/private-services.md),
    the private Knative Service will be accessible from public internet with the custom domain of the domain mapping.

```

Reply:
In the preview, I noticed that the body text for the note is appearing outside the note box.

Screenshot of issue:
<img width="727" alt="Screen Shot 2021-06-04 at 14 19 34" src="https://user-images.githubusercontent.com/33425661/120807499-ec1f0700-c53f-11eb-9dc2-81d28fe3eb55.png">
My suggestion above adds spaces to fix this.

Reply:
Done